assets/src/application/ui/forms/espressoForm/EspressoForm.tsx

```diff
@@ -10,18 +10,15 @@ import './styles.scss';
 
 const EspressoForm: React.FC<EspressoFormProps> = ({ onSubmit, mutators, layout, ...rest }) => {
 	const context = { layout };
+	const formMutators = {
+		...arrayMutators,
+		...mutators,
+		updateFieldValue,
+	};
+
 	return (
 		<FormProvider value={context}>
-			<ReactFinalForm
-				onSubmit={onSubmit}
-				render={FormRenderer}
-				mutators={{
-					...arrayMutators,
-					...mutators,
-					updateFieldValue,
-				}}
-				{...rest}
-			/>
+			<ReactFinalForm component={FormRenderer} onSubmit={onSubmit} mutators={formMutators} {...rest} />
```

Comment:
is it safe to pass the rest of the props as last argument here?

Reply:
Yes, consumer can have full control